package.json

```diff
@@ -40,7 +40,7 @@
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
     "bower-config": "^1.3.0",
     "bower-endpoint-parser": "0.2.2",
-    "broccoli-babel-transpiler": "6.0.0-alpha.3",
+    "broccoli-babel-transpiler": "6.0.0",
```

Comment:
Can you swap this to ^6.0.0? Also, could you update the target branch to be `beta`?